docs/release_notes.md

```diff
@@ -5,6 +5,102 @@ hide:
+- [[PR #10800]](https://github.com/kubevirt/kubevirt/pull/10800) [AlonaKaplan] Support macvtap as a binding plugin
+- [[PR #10425]](https://github.com/kubevirt/kubevirt/pull/10425) [ormergi] Introduce network binding plugin for Passt networking, interfacing with Kubevirt new network binding plugin API.
+
+### SIG-infra
+- [[PR #11025]](https://github.com/kubevirt/kubevirt/pull/11025) [0xFelix] Allow unprivileged users read-only access to VirtualMachineCluster{Instancetypes,Preferences} by default.
+- [[PR #10922]](https://github.com/kubevirt/kubevirt/pull/10922) [kubevirt-bot] Updated common-instancetypes bundles to v0.4.0
+
+### SIG-scale
+- [[PR #10571]](https://github.com/kubevirt/kubevirt/pull/10571) [tiraboschi] vmi memory footprint increase by 35M when guest serial console logging is turned on (default on).
+
+### Monitoring
+- [[PR #10982]](https://github.com/kubevirt/kubevirt/pull/10982) [machadovilaca] Refactor monitoring metrics
+- [[PR #10962]](https://github.com/kubevirt/kubevirt/pull/10962) [machadovilaca] Update monitoring file structure
+- [[PR #10853]](https://github.com/kubevirt/kubevirt/pull/10853) [machadovilaca] Refactor monitoring collectors
+- [[PR #10700]](https://github.com/kubevirt/kubevirt/pull/10700) [machadovilaca] Refactor monitoring alerts
+- [[PR #10693]](https://github.com/kubevirt/kubevirt/pull/10693) [machadovilaca] Remove MigrateVmiDiskTransferRateMetric
+- [[PR #10651]](https://github.com/kubevirt/kubevirt/pull/10651) [machadovilaca] Refactor monitoring  recording-rules
+- [[PR #10570]](https://github.com/kubevirt/kubevirt/pull/10570) [machadovilaca] Fix LowKVMNodesCount not firing
+- [[PR #10418]](https://github.com/kubevirt/kubevirt/pull/10418) [machadovilaca] Add total VMs created metric
+
+### Uncategorized
+- [[PR #11144]](https://github.com/kubevirt/kubevirt/pull/11144) [0xFelix] virtctl: Specifying size when creating a VM and using --volume-import to clone a PVC or a VolumeSnapshot is optional now
+- [[PR #11122]](https://github.com/kubevirt/kubevirt/pull/11122) [brianmcarey] Update runc dependency to v1.1.12
+- [[PR #11068]](https://github.com/kubevirt/kubevirt/pull/11068) [brianmcarey] Update container base image to use current stable debian 12 base
+- [[PR #11050]](https://github.com/kubevirt/kubevirt/pull/11050) [fossedihelm] restrict default cluster role to authenticated only users
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
```

Comment:
bugfix?

Reply:
I've moved it as suggested but: Is it a bugfix if there's no bug? What is the sound of one hand clapping? :)